Sources/Controllers/Base/OABaseNavbarSubviewViewController.m

```diff
@@ -47,16 +47,16 @@ - (void)setupContainerAppearance:(UIView *)container
     switch ([self getNavbarColorScheme])
     {
         case EOABaseNavbarColorSchemeGray:
-            container.backgroundColor = self.tableView.backgroundColor;
+            container.backgroundColor = UIColor.viewBgColor;
             break;
         case EOABaseNavbarColorSchemeOrange:
-            container.backgroundColor = UIColorFromRGB(color_osmand_orange);
+            container.backgroundColor = [UIColor.navBarBgColorPrimary colorWithAlphaComponent:1.0];
```

Comment:
colorWithAlphaComponent ?

Reply:
I double-checked, everything is correct!